ephypype/preproc.py

```diff
@@ -248,7 +248,7 @@ def get_raw_sfreq(raw_fname):
 
     try:
         data = mne.io.read_raw_fif(raw_fname)
-    except:
+    except:  # noqa
```

Comment:
The function `get_raw_sfreq` is not used in the package... Anyway, it should be better to catch the error type raise by MNE instead of a noqa